app/controllers/games_controller.rb

```diff
@@ -1,2 +1,30 @@
 class GamesController < ApplicationController
+
+	def move_piece(piece, position2)
+		# piece is the current piece being moved (belongs to current user and current game) and it has a current position
+		# this method could be adjusted to take in x_axis & y_axis parameters instead of position2
+
```

Comment:
Solid work, Denis.

Can you move the bulk of this code to the game model and test it there:

```
def is_obstructed?(position_1, position_2)

end
```

I also think in the model `position_1` and `position_2` should be x,y pairs.  Like `[0,3]`, instead of string values that we get from the controller.